### PR TITLE
Added Information About Deleting Admin Users

### PR DESCRIPTION
**Pull request by DmitryTsymbal**
## Purpose of this pull request

This PR provides an information about deleting admin users

## Magento release version

_Which Magento release(s) are affected by the content changes: 2.4, 2.3? Specify a patch release number, if applicable._

## Product editions

_Is this update specific to a product edition: Magento Open Source only, Adobe Commerce only?_

- ...

_Is this update specific to an installed feature extension: B2B extension, other feature set (please, specify)?_

- ...

## Affected documentation pages

https://docs.magento.com/user-guide/system/permissions-users-all.html

### CODE REVIEW

**Comment by DmitryTsymbal** [comment]
@magento run all tests

File: src/system/permissions-users-all.md [hunk 1]
```diff
@@ -85,3 +85,15 @@ _Example Admin Users_
 1. Click <span class="btn">Save User</span>. When prompted, enter **Your Password** to confirm your identity, and again click <span class="btn">Save User</span>.
 
    The _Users_ grid opens and lists all users.
+
+## Deleting Admin Users
```
Comment (leslietilling) [inline src/system/permissions-users-all.md:89]:
```suggestion
## Delete an Admin user
```

File: src/system/permissions-users-all.md [hunk 2]
```diff
@@ -85,3 +85,15 @@ _Example Admin Users_
...
+
+1. Navigate to admin users grid: **System** > _Permissions_ > **All Users**.
```
Comment (leslietilling) [inline src/system/permissions-users-all.md:91]:
```suggestion
1. On the _Admin_ sidebar, go to **System** > _Permissions_ > **All Users**.
```

File: src/system/permissions-users-all.md [hunk 3]
```diff
@@ -85,3 +85,15 @@ _Example Admin Users_
...
+
+1. Find needed user account using filters on grid and click user name.
```
Comment (leslietilling) [inline src/system/permissions-users-all.md:93]:
```suggestion
1. Locate the user account using filters above the grid and click the user name.
```

File: src/system/permissions-users-all.md [hunk 4]
```diff
@@ -85,3 +85,15 @@ _Example Admin Users_
...
+
+1. Enter the current admin' user password into **Your Password** field.
```
Comment (leslietilling) [inline src/system/permissions-users-all.md:95]:
```suggestion
1. When prompted, enter **Your Password** to confirm your identity.
```

File: src/system/permissions-users-all.md [hunk 5]
```diff
@@ -85,3 +85,15 @@ _Example Admin Users_
...
+
+1. In the upper-right corner, click <span class="btn">Delete User</span>.
+
+1. Confirm the action by clicking <span class="btn">OK</span> button.
```
Comment (leslietilling) [inline src/system/permissions-users-all.md:99]:
```suggestion
1. To confirm the action, click <span class="btn">OK</span>.
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
Edits for style and clarity

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@leslietilling the pull request successfully imported.

**Pull request merged**